sync-customerio/extension.yaml

```diff
@@ -210,8 +154,5 @@ params:
     description: A collection path to monitor for changes
     type: string
     example: users
-    # ideally we want this to be optional however if unset then see errors
-    # and slowdowns inserting into firestore so set default value to an unsed
-    # collection name.
```

Comment:
we should probably update the description and mention that setting an unused collection name is a way to disable this function